catalyst_voices/lib/pages/registration/create_keychain/stage/unlock_password_panel.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:catalyst_voices/pages/registration/registration_stage_navigation.dart';
+import 'package:catalyst_voices/widgets/indicators/voices_password_strength_indicator.dart';
+import 'package:catalyst_voices/widgets/text_field/voices_password_text_field.dart';
+import 'package:catalyst_voices/widgets/text_field/voices_text_field.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+class UnlockPasswordPanel extends StatefulWidget {
+  final UnlockPasswordState data;
+
+  const UnlockPasswordPanel({
+    super.key,
+    required this.data,
+  });
+
+  @override
+  State<UnlockPasswordPanel> createState() => _UnlockPasswordPanelState();
+}
+
+class _UnlockPasswordPanelState extends State<UnlockPasswordPanel> {
+  late final TextEditingController _passwordController;
+  late final TextEditingController _confirmPasswordController;
+
+  @override
+  void initState() {
+    super.initState();
+
+    final registrationState = RegistrationCubit.of(context).state;
+    final createKeychain =
```

Comment:
Shall we suffix it with `State` for clarity?

Reply:
I'm going to refactor it soon in separate PR but yes, ill need to make it consistent.